tests/components/RxView/rx-view.test.js

```diff
@@ -172,40 +172,6 @@ describe('RxView component', () => {
     expect(updateDate).toHaveBeenCalled();
   });
 
-  it('updates the FHIR resource and calls CDS Services if a prescription is selected', async (done) => {
```

Comment:
The RxView component is no longer responsible for directly triggering service calls. (TODO: consider adding new tests on the middlewares that do trigger these service calls.)

Reply:
Great! To start, I'm adding middleware tests to make sure the handlers fire on:

* postMessage
* system actions
* resource context changes

Reply:
Also added a singe test as a starting point on the Pama component.